### PR TITLE
1.2: Update the cli command reference documentation directory

### PR DESCRIPTION
**Pull request by Caleb-Hill**
Updates to cli command reference documentation directory.

### CODE REVIEW

File: docs/core/1.2/cli/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# CLI Command Reference
```
Comment (vaporos) [inline docs/core/1.2/cli/index.md]: Duplicate header

File: docs/core/1.2/cli/sawadm.md [hunk 2]
```diff
@@ -35,34 +51,23 @@ initialization of a validator. A network requires an initial block
...
-(Use a space to separate multiple files.) If no input file is specified,
-this command produces an empty genesis block. ======= The
-[input_file]{.title-ref} argument specifies one or more files containing
-serialized `BatchList` protobuf messages to add to the genesis data.
-(Use a space to separate multiple files.) At least one input file must
-be specified and it must contain the required settings, unless the
-`--ignore-required-settings` argument is used (run
-`sawadm genesis --help` for more info). \>\>\>\>\>\>\>
-core/1-2:docs/core/1.2/cli/sawadm.rst
+The [input_file]{.title-ref} argument specifies one or more files containing
```
Comment (vaporos) [inline docs/core/1.2/cli/sawadm.md]: .title-ref is a sphinx-ism I think

**Review by vaporos — CHANGES_REQUESTED** [review]
Needs to be moved an hooked up the the sidebar.

File: docs/core/1.2/cli/index.md (hunk 1 above)
Comment (Caleb-Hill) [reply docs/core/1.2/cli/index.md]: Fixed

File: docs/core/1.2/cli/sawadm.md (hunk 2 above)
Comment (Caleb-Hill) [reply docs/core/1.2/cli/sawadm.md]: Fixed in all files I had already worked in

File: docs/core/1.2/cli/sawtooth-validator.md [hunk 3]
```diff
@@ -35,9 +35,9 @@ validator.
     network permissions, specify `challenge`, which requires connections
     to sign a challenge so their identity can be proved.
 
-::: literalinclude
-output/sawtooth-validator_usage.out
-:::
+``` console
```
Comment (rbuysse) [inline docs/core/1.2/cli/sawtooth-validator.md]: Looks like this got missed

```
sawtooth-validator 1.2.6
Configures and starts a Sawtooth validator.

USAGE:
    sawtooth-validator [FLAGS] [OPTIONS]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information
    -v, --verbose    enable more verbose output to stderr

OPTIONS:
    -B, --bind <bind>...
            set the URL for the network or validator component service endpoints with the format network:<endpoint>,
            component:<endpoint>, or consensus:<endpoint>. Use multiple --bind options to specify all endpoints.
        --config-dir <config_dir>                                  specify the configuration directory
    -E, --endpoint <endpoint>                                      specifies the advertised network endpoint URL
        --fork-cache-keep-time <fork_cache_keep_time>              set the time in seconds to keep uncommitted forks.
        --maximum-peer-connectivity <maximum_peer_connectivity>    set the maximum number of peers to accept
        --minimum-peer-connectivity <minimum_peer_connectivity>
            set the minimum number of peers required before stopping peer search

        --network-auth <network_auth>
            identify type of authorization required to join validator network. [possible values: trust, challenge]

        --opentsdb-db <opentsdb-db>                                specify name of database used for storing metrics
        --opentsdb-url <opentsdb-url>
            specify host and port for Open TSDB database used for metrics

    -P, --peering <peering>
            determine peering type for the validator: 'static' (must use --peers to list peers) or 'dynamic' (processes
            any static peers first, then starts topology buildout). [possible values: static, dynamic]
    -p, --peers <peers>...
            list static peers to attempt to connect to in the format tcp://<hostname>:<port>. Specify multiple peers in
            a comma-separated list. Repeating the --peers option is also accepted.
        --scheduler <scheduler>
            set scheduler type: serial or parallel [possible values: serial, parallel]

    -s, --seeds <seeds>...
            provide URI(s) for the initial connection to the validator network, in the format tcp://<hostname>:<port>.
            Specify multiple URIs in a comma-separated list. Repeating the --seeds option is also accepted.
        --state-pruning-block-depth <state_pruning_block_depth>
            set the block depth below which state roots are pruned from the global state database.
```

**Review by vaporos — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Pull request merged**